runtime/tests/mock.rs

```diff
@@ -93,14 +322,14 @@ impl ExtBuilder {
 	}
 	pub fn build(self) -> sp_io::TestExternalities {
 		let mut endowed_accounts = vec![alice(), bob(), charlie(), dave(), eve(), ferdie()];
-		let initial_authorities = generate_initial_authorities(self.validator_count);
-		let stash_accounts: Vec<_> = initial_authorities.iter().map(|x| x.0.clone()).collect();
+		let initial_validators = validators(self.validator_count);
+		let stash_accounts: Vec<_> = initial_validators.iter().map(|x| x.0.clone()).collect();
 		endowed_accounts.extend(stash_accounts);
 
 		let mut t = frame_system::GenesisConfig::default()
-			.build_storage::<Runtime>()
+			.build_storage::<Test>()
```

Comment:
Something to note is: by passing in Test instead of Runtime, we are no longer testing the actual Runtime struct <- which was the original purpose I think

Reply:
Had a discussion offline. We will keep using `Test` until we can correctly trigger session end based on Babe